forge.config.cjs

```diff
@@ -17,33 +17,37 @@ module.exports = {
         overwrite: true,
         asar: true,
         icon: "./images/app-icons/icon",
+        osxSign: {},
+        osxNotarize: {
+            appleId: process.env.APPLE_ID,
+            appleIdPassword: process.env.APPLE_ID_PASSWORD,
+            teamId: process.env.APPLE_TEAM_ID
+        },
         extraResource: [
-            // Moved to root
-            ...extraResourcesForPlatform,
+            // All resources should stay in Resources directory for macOS
+            ...(process.platform === "darwin" ? [] : extraResourcesForPlatform),
 
-            // Moved to resources (TriliumNext Notes.app/Contents/Resources on macOS)
+            // These always go in Resources
             "translations/",
             "node_modules/@highlightjs/cdn-assets/styles"
         ],
         afterComplete: [
             (buildPath, _electronVersion, platform, _arch, callback) => {
-                for (const resource of extraResourcesForPlatform) {
-                    const baseName = path.basename(resource);
-
-                    // prettier-ignore
-                    const sourcePath = (platform === "darwin")
-                        ? path.join(buildPath, `${APP_NAME}.app`, "Contents", "Resources", baseName)
-                        : path.join(buildPath, "resources", baseName);
-
-                    // prettier-ignore
```

Comment:
could you add that comment back in above `destPath` please?

Reply:
I believe I put it back where you asked, now 👀